app.js

```diff
@@ -71,29 +71,29 @@ try {
  *********************************************************/
 
 try {
-	require.resolve('./config/config.js');
+	require.resolve('./config/config');
 } catch (err) {
 	if (err.code !== 'MODULE_NOT_FOUND') throw err; // should never happen
 
 	// Copy it over synchronously from config-example.js since it's needed before we can start the server
-	console.log("config.js doesn't exist - creating one with default settings...");
+	console.log("config doesn't exist - creating config.js with default settings...");
 	fs.writeFileSync(path.resolve(__dirname, 'config/config.js'),
 		fs.readFileSync(path.resolve(__dirname, 'config/config-example.js'))
 	);
 } finally {
-	global.Config = require('./config/config.js');
+	global.Config = require('./config/config');
 }
 
 if (Config.watchconfig) {
-	fs.watchFile(path.resolve(__dirname, 'config/config.js'), (curr, prev) => {
+	fs.watchFile(path.resolve(__dirname, 'config/config'), (curr, prev) => {
```

Comment:
Keep the extension here.

Or, if you want to deal with this in a extension-agnostic way, pass the resolved module path